__tests__/components/screens/ScreenHomeAdmin.test.tsx

```diff
@@ -7,10 +8,25 @@ import { ScreenHomeAdmin } from "../../../src/components/screens/ScreenHomeAdmin
 
 console.log = jest.fn();
 
+jest.mock("../../../src/constants/Constants", () => {
+    const mockFunctionOutput = {
+        apiUrl: "http://localhost:8080/",
+        authEndpoint: "http://keycloak:9080/auth/realms/jhipster/protocol/openid-connect/auth",
+        authTokenEndpoint: "http://keycloak:9080/auth/realms/jhipster/protocol/openid-connect/token",
+        channel: "dev",
+    };
+
+    return {
+        itRexVars: jest.fn(() => {
+            return mockFunctionOutput;
+        }),
+    };
+});
+
 describe("test ScreenHomeAdmin", () => {
     it("see if stuff is rendered", () => {
         const { getByText } = render(<ScreenHomeAdmin></ScreenHomeAdmin>);
 
-        expect(getByText("Admin Home")).toBeDefined();
```

Comment:
I believe this is fine; the left one looks "cleaner" to me so one might assume that that is more up-to-date, but I think I've seen the string on the right during my hasty tests so scream if this is wrong.